engine/flink/util/src/main/scala/pl/touk/nussknacker/engine/flink/util/metrics/FlinkMetricsProviderForScenario.scala

```diff
@@ -55,6 +53,10 @@ class FlinkMetricsProviderForScenario(runtimeContext: RuntimeContext) extends Me
     group.histogram(name, histogram)
   }
 
+  override def remove(metricIdentifier: MetricIdentifier): Unit = {
+    // Not needed handling of situation of metrics removal
```

Comment:
Let's throw NotImplementedException or sth, in case sb wants to actually use it ;)